crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -644,19 +645,19 @@ impl<B: BlockT> futures::Future for SubspaceWorker<B> {
     }
 }
 
-struct SubspaceSlotWorker<B: BlockT, C, E, I, SO, L, BS> {
-    client: Arc<C>,
-    block_import: I,
-    env: E,
-    sync_oracle: SO,
-    justification_sync_link: L,
-    force_authoring: bool,
-    backoff_authoring_blocks: Option<BS>,
-    subspace_link: SubspaceLink<B>,
-    signing_context: SigningContext,
-    block_proposal_slot_portion: SlotProportion,
-    max_block_proposal_slot_portion: Option<SlotProportion>,
-    telemetry: Option<TelemetryHandle>,
+pub(crate) struct SubspaceSlotWorker<B: BlockT, C, E, I, SO, L, BS> {
```

Comment:
Why not moving `SubspaceSlotWorker` into separate module then?

Reply:
Done in https://github.com/subspace/subspace/pull/99/commits/bd5eb276447f2030be27bf2b98a0762eb49eb2be . Was wondering if it's good also to separate out `Verifier` given that we already have `verification.rs`.